effekt/shared/src/main/scala/effekt/core/Renamer.scala

```diff
@@ -4,7 +4,8 @@ import effekt.{ core, symbols }
 import effekt.context.Context
 
 /**
- * Freshens bound names in a given term
+ * Freshens bound names in a given Core term.
+ * Do not use for tests!
```

Comment:
```suggestion
 * Freshens bound names in a given Core term.
 * Do not use for tests! See [[effekt.core.TestRenamer]].
```
Maybe make this more actionable.